src/plugins/sentry/index.ts

```diff
@@ -0,0 +1,10 @@
+import { init } from '@sentry/node'
+import { nodeProfilingIntegration } from '@sentry/profiling-node'
+
+export function initSentry(dsn: string): void {
+  init({
+    dsn,
```

Comment:
question: we'll have multiple monikas running, how do we know wich monika generated the error? One differenct dsn for different monika? or one dsn for all of them?

if one dsn for all of them maybe add a release: info to have something like "monika-capetown@1.2.3" for identification?

Reply:
I've added more data to identify

```js
    if (getContext().flags.sentryDSN !== undefined) {
      // Get current public IP
      let publicIp
      const connection = await sendPing('stun.l.google.com')
      if (connection.alive) {
        const response = await stun.request('stun.l.google.com:19302')
        publicIp = response?.getXorAddress()?.address
      }

      const response = await sendHttpRequest({
        url: `http://ip-api.com/json/${publicIp}`,
      })
      const { country, city, isp } = response.data

      const tags: {
        country: string
        city: string
        hostname: string
        isp: string
        privateIp: string
        publicIp: string
        monikaVersion: string
        symonMonikaId?: string
      } = {
        country,
        city,
        hostname: hostname(),
        isp,
        privateIp: getIp(),
        publicIp,
        monikaVersion: this.config.version,
      }

      if (getContext().flags.symonMonikaId) {
        tags.symonMonikaId = getContext().flags.symonMonikaId
      }

      captureException(error, {
        tags,
      })
    }
```

Though it is unlikely that Sentry will be used for

Reply:
![image](https://github.com/user-attachments/assets/988c6248-8131-40af-841f-2da229316d52)

Added monikaVersion, publicIp, and locations